reason-react-native/src/apis/CameraRoll.md

```diff
@@ -130,7 +131,95 @@ type action =
   | SetPath(string)
   | SetPhotos(array(CameraRoll.photoIdentifier));
 
-let textStyle = Style.style(~textAlign=`center, ());
+let styles =
+  StyleSheet.create({
+    "container":
+      Style.style(
+        ~flex=1.,
+        ~flexDirection=`column,
+        ~alignItems=`center,
+        ~justifyContent=`spaceBetween,
+        (),
+      ),
+    "getPhotosExample":
+      Style.style(
```

Comment:
I'd open Style before StyleSheet.create:

```re
let styles =
	  Style.(StyleSheet.create({
	    "container":
	     style(
	     ...
```

Then you can just write `style(...)` and `pt(...)` without the `Style.`.

Reply:
Good suggestion, I'm familiar with local opens, but don't use them often. Will update the example to take advantage of that.